migrations/20240613102553-add-accounting-category-applies-to.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn('AccountingCategories', 'appliesTo', {
+      type: Sequelize.ENUM(['HOST', 'HOSTED_COLLECTIVES']),
+      allowNull: false,
+      defaultValue: 'HOSTED_COLLECTIVES',
+    });
+
+    await queryInterface.sequelize.query(`
+      UPDATE "AccountingCategories"
+      SET "appliesTo" = 'HOSTED_COLLECTIVES'
+      WHERE "CollectiveId" IN (
+        SELECT id FROM "Collectives" where type = 'ORGANIZATION'
+      );
+    `);
```

Comment:
This one doesn't seem necessary, given the `defaultValue` set above will already be applied to all rows?

Reply:
Yes